base/libgit2/libgit2.jl

```diff
@@ -458,6 +458,23 @@ Equivalent to `git checkout [-f] --detach <commit>`.
 Checkout the git commit `commit` (a [`GitHash`](@ref) in string form)
 in `repo`. If `force` is `true`, force the checkout and discard any
 current changes. Note that this detaches the current HEAD.
+
+# Example
+
+```julia
+repo = LibGit2.init(repo_path)
+open(joinpath(LibGit2.path(repo),"file1"),"w") do f
```

Comment:
For consistency, perhaps inject spaces between function arguments in this line and the similar `open` call below?